gen/calc.py

```diff
@@ -200,6 +200,16 @@ def validate_mesos_container_log_sink(mesos_container_log_sink):
         "Container logs must go to 'journald', 'logrotate', or 'journald+logrotate'."
 
 
+def validate_metronome_gpu_scheduling_behavior(metronome_gpu_scheduling_behavior):
```

Comment:
I don't see these methods being used.

Reply:
it is a good question.   I was assuming that `validate_`variable was auto trigged.  the `validate_mesos_container_log_sink` above this function either does that or isn't involved either:)    there is a `validate` entry in `entry` object as well though.   It isn't clear if registration in that is required.   https://github.com/dcos/dcos/blob/master/gen/calc.py#L1009

@orsenthil do you know?

Reply:
@kensipe it seems like they do need to be included in the 'validate' array:

https://github.com/dcos/dcos/blob/1fb126d8d904d7ca06f6eef17af886e42856e914/gen/calc.py#L1019

IE

```
/Users/tim/src/m8e/dcos/gen/calc.py

 626 | def validate_dns_forward_zones(dns_forward_zones):
 627 |     """
 628 |      "forward_zones": {"a.contoso.com": ["1.1.1.1:53", "2.2.2.2"],
/Users/tim/src/m8e/dcos/gen/calc.py

1018 | entry = {
1019 |     'validate': [
...
1027 |         validate_dns_bind_ip_blacklist,
1028 |         validate_dns_forward_zones,
1029 |         validate_zk_hosts,
```